e2e/test-component/scenarios/embedding-sdk/editable-dashboard.cy.spec.tsx

```diff
@@ -104,4 +115,77 @@ describeEE("scenarios > embedding-sdk > editable-dashboard", () => {
       });
     });
   });
+
+  describe("EMB-84", () => {
+    beforeEach(() => {
+      cy.signInAsAdmin();
+      cy.log("Create a dashboard with Created at filter with a default value");
```

Comment:
I create a new dashboard inside the new test in this suite because the existing dashboard doesn't contain any card. Unlike the other test suite, the dashboard already contain a card, so I only map it to a newly added filter.